.github/actions/setup-sentry/action.yml

```diff
@@ -44,7 +44,8 @@ runs:
         echo "SENTRY_LIGHT_BUILD=1" >> $GITHUB_ENV
         echo "SENTRY_SKIP_BACKEND_VALIDATION=1" >> $GITHUB_ENV
 
-        [ "$PYTHON_VERSION" = "3" ] && echo "SENTRY_PYTHON3=1" >> $GITHUB_ENV || true
+        [ "$PYTHON_VERSION" = "2" ] && echo "SENTRY_PYTHON2=1" >> $GITHUB_ENV || true
+        [ "$PYTHON_VERSION" = "2.7.17" ] && echo "SENTRY_PYTHON2=1" >> $GITHUB_ENV || true
```

Comment:
I'm unsure if both are needed but we will remove this in December when we drop Python 2 completely.

Reply:
It's likely we won't drop python 2 until the beginning of the new year.

The current plan is that we'll deploy on the 2nd, and wait about a month before we start ripping out python 2 code. The reason is that we don't want to get into a place where something unexpected happens and we _need_ to roll back and we're in a place where we cant.

Reply:
Got it.